content/_index.adoc

```diff
@@ -1,8 +1,8 @@
 ---
 title: "AsciiDoc Working Group"
 seo_title: "AsciiDoc Working Group"
-headline: "AsciiDoc Working Group"
-tagline: "We drive the standardization, evolution, and adoption of the AsciiDoc language through open collaboration and open source."
+headline: <h1>AsciiDoc<sup>®</sup> Working Group</h1>
```

Comment:
@linkfang I am looking at our jumbotron template and it appears as we are duplicating the h1 and p tag here: 
https://github.com/EclipseFdn/hugo-solstice-theme/blob/master/layouts/partials/jumbotron.html#L19

Can you remove here.

Reply:
Sure, good catch!